core/src/blocktree.rs

```diff
@@ -2792,15 +2753,14 @@ pub mod tests {
                         // between 1-4 data blobs are missing
                         if rng.gen() {
                             blocktree
-                                .write_shared_blobs(erasure_set.data)
+                                .write_blobs(erasure_set.data)
```

Comment:
write_blobs() move them, yes?

Reply:
It doesn't, it uses `Borrow` and only uses a reference to blobs